packages/components/src/index.ts

```diff
@@ -128,6 +128,7 @@ export { default as PanelHeader } from './panel/header';
 export { default as PanelRow } from './panel/row';
 export { default as Placeholder } from './placeholder';
 export { default as Popover } from './popover';
+export { default as __experimentalProgressBar } from './progress-bar';
```

Comment:
We should make this private and name it without the prefix.

Reply:
Of course. Done in https://github.com/WordPress/gutenberg/pull/53030/commits/cafb1d86f78ff2937783a07010b69efdfc41500a